MAUI/Carousel-View/LoadMore.md

```diff
@@ -0,0 +1,517 @@
+
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <DataTemplate x:Key="itemTemplate">
+                <Image Source="{Binding Image}"
+				Aspect="AspectFit"/>
+            </DataTemplate>
+        </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <ContentPage.Content>
+        <carousel:SfCarousel x:Name="carousel"
+			ItemTemplate="{StaticResource itemTemplate}"
+			ItemsSource="{Binding ImageCollection}"
+			ItemHeight="200"
+			ItemWidth="200"
+			ItemSpacing="2"
+			AllowLoadMore="True"
+			ViewMode="Linear">
+        </carousel:SfCarousel>
+    </ContentPage.Content>
+</ContentPage>
+	
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Carousel;
+using System.Collections.Generic;
+
+namespace CarouselSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            this.BindingContext = new CarouselViewModel();
+            SfCarousel carousel = new SfCarousel()
+            {
+                ItemHeight = 200,
+                ItemWidth = 200,
+                ItemSpacing = 2,
+                AllowLoadMore = true,
+                ViewMode = ViewMode.Linear
+            };
+
+            carousel.ItemTemplate = new DataTemplate(() =>
+            {
+                Image image = new Image();
+                image.SetBinding(Image.SourceProperty, "Image");
+                image.Aspect = Aspect.AspectFit;
+                return image;
+            });
+
+            carousel.SetBinding(SfCarousel.ItemsSourceProperty, "ImageCollection");
+            this.Content = carousel;
+        }
+    }
+
+    public class CarouselModel
+    {
+        public CarouselModel(string imageString)
+        {
+            Image = imageString;
+        }
+        private string _image;
+
+        public string Image
+        {
+            get { return _image; }
+            set { _image = value; }
+        }
+    }
+
+    public class CarouselViewModel
+    {
+        public CarouselViewModel()
+        {
+            ImageCollection.Add(new CarouselModel("carousel_person1.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person2.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person3.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person4.png"));
+            ImageCollection.Add(new CarouselModel("carousel_person5.png"));
+        }
+
+        private List<CarouselModel> imageCollection = new List<CarouselModel>();
+        public List<CarouselModel> ImageCollection
+        {
+            get { return imageCollection; }
+            set { imageCollection = value; }
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## LoadMoreItemsCount
+
+Number of items can be maintained in the carousel control by using the `LoadMoreItemsCount` property. By using the `LoadMoreItemsCount` property, numerous items can be separated. 
```

Comment:
Number of items can be maintained in the carousel control using the `LoadMoreItemsCount` property. By using the `LoadMoreItemsCount` property, numerous items can be separated.

Reply:
Addressed the changes.